trade-finance-manager-ui/server/controllers/case/amendments/amendCoverEndDate.test.js

```diff
@@ -1,8 +1,9 @@
-import { isTfmFacilityEndDateFeatureFlagEnabled, TEAM_IDS } from '@ukef/dtfs2-common';
+import { FACILITY_TYPE, isTfmFacilityEndDateFeatureFlagEnabled, TEAM_IDS } from '@ukef/dtfs2-common';
 import api from '../../../api';
 import { mockRes } from '../../../test-mocks';
 import { MOCK_AMENDMENT_COVERENDDATE_CHANGE, MOCK_AMENDMENT_FACILITYVALUE_AND_COVERENDDATE_CHANGE } from '../../../test-mocks/amendment-test-mocks';
```

Comment:
can the names of these mocks be changed? eg COVER_END_DATE, FACILITY_VALUE

Reply:
I think it's important to make it clear that it's an amendment with a cover end date change, as opposed to just a cover end date. So I think the current names while long are clearer.

Reply:
Sorry I mean that the full thing should be UPPER_SNAKE_CASE including COVERENDDATE to COVER_END_DATE in between etc

Reply:
Ah I understand, this is a legacy issue throughout (and not introduced in this PR):
<img width="484" alt="image" src="https://github.com/user-attachments/assets/50a4a787-86ca-4f16-80da-5990ad772102">
So it's out of scope to change it here.